dproc/dproc.go

```diff
@@ -41,8 +42,77 @@ type DataprocRequestParams struct {
 	Location      string
 }
 
-func GetDataprocRequestParams(sourceProfile profiles.SourceProfile, targetProfile profiles.TargetProfile, srcSchema string, srcTable string, primaryKeys string, location string, subnet string) (DataprocRequestParams, error) {
+func getSparkType(spColType string) string {
+	switch spColType {
+	case "INT64":
+		return "LONG"
+	case "FLOAT64":
+		return "DOUBLE"
+	case "BYTES":
+		return "BINARY"
+	case "BOOL":
+		return "BOOLEAN"
+	case "STRING":
+		return "STRING"
+	case "TIMESTAMP":
+		return "TIMESTAMP"
+	case "DATE":
+		return "DATE"
+	case "NUMERIC":
+		return "DECIMAL"
+	default:
```

Comment:
JSON type ?

Reply:
Currently complex datatypes (like JSON, ARRAY, SET) are not supported by the dataproc template